### ISSUE TITLE
export * from ... fails on compile time

### ISSUE THREAD

**Issue by DocX** [davezuko/react-redux-starter-kit#1133]
Add file with this content to project with default `react-redux-starter-kit` configuration:

```
export * from './anotherFile.js';
```

content of `anotherFile` is irrelevant.

What happens:

* Get runtime error `Uncaught SyntaxError: Unexpected token import`
* Investigating compiled code with error, this is what the module gets compiled to:

```
	'use strict';
	
	import _Object$defineProperty from 'babel-runtime/core-js/object/define-property';
	import _Object$keys from 'babel-runtime/core-js/object/keys';
	Object.defineProperty(exports, "__esModule", {
	  value: true
	});
	
	var _previews = __webpack_require__(776);
	
	_Object$keys(_previews).forEach(function (key) {
	  if (key === "default" || key === "__esModule") return;
	
	  _Object$defineProperty(exports, key, {
	    enumerable: true,
	    get: function get() {
	      return _previews[key];
	    }
	  });
	});

```

What is expected:

* It should JustWork(tm)
* Perhaps `import` should get compiled to WebPack's `require`. I assume it doesn't get compiled because this is already result of compilation? :/

I am not sure if the issue is in WebPack, Babel, or configuration of react-redux-starter-kit

Any ideas?
Thank you

**Comment by davezuko** [comment]
Yeah that _really_ should not be happening. How webpack 1 w/ ES6 modules works is that babel transpiles them down to CommonJS, and then webpack understands how to resolve those. Webpack 2 natively understands ES6 modules so there's no transpilation necessary. We should upgrade soon, but this still should not be happening; will try to repro.

**Comment by DocX** [comment]
Thanks.

Looking around i found this bug reported in babel: https://github.com/babel/babel/issues/2877

They suggest turning off the polyfill option of runtime (which is the cause - it sees `Object.defineProperty` and wants to import the polyfill):

```
"plugins": [
    ["transform-runtime", { "polyfill": false }]
  ]
```
